src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -158,32 +205,53 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     ) {
       applyThemesOnElement(this, this.hass.themes, this._config.theme);
     }
+
+    const oldImage = oldHass
+      ? oldHass.states[this._config.entity].attributes.entity_picture
+      : undefined;
+    const newImage = this.hass.states[this._config.entity].attributes
+      .entity_picture;
+
+    if (!newImage || newImage === oldImage) {
+      return;
+    }
+
+    if (newImage.substr(0, 1) !== "/") {
+      this._image = newImage;
+      return;
+    }
+
+    fetchMediaPlayerThumbnailWithCache(this.hass, this._config.entity).then(
```

Comment:
@balloob didn't we move the thumbnails away from the ws?

Reply:
What is the status on how we should fetch the image? Should it be reverted to using the image URL or is this the best method. This is the method it was using before the conversion.